core/src/com/mygdx/game/models/Orders.java

```diff
@@ -0,0 +1,58 @@
+package com.mygdx.game.models;
+
+
+import java.util.ArrayList;
+
+
+/**
+ * All of the Commands given to a unit for a turn
+ */
+public class Orders {
+
+    /* The acting unit's ID */
+    private int unitId;
+
+    /* The number of Commands in the Orders */
+    private int length;
+
+    /* The Commands the Orders contain */
+    private ArrayList<Command> commands;
+
+    /* The current Command*/
+    private Command currentCommand;
+
+    /* The index of the current Command */
+    private int currentCommandIndex;
+
+    public Orders(int unitId, int length, ArrayList<Command> commands) {
+        this.unitId = unitId;
+        this.length = length;
+        this.commands = commands;
+    }
+
+    public Command getCurrentCommand() {
+        return currentCommand;
+    }
+
+    /* Iterates to the next Command */
+    public void nextCommand() {
```

Comment:
I feel like this should return command

Reply:
Done

Reply:
Changed to return a boolean of whether there is a next Command, and doesn't iterate if there isn't one.